timber/src/main/java/timber/log/Timber.java

```diff
@@ -391,14 +391,24 @@ private Timber() {
   public static abstract class Tree {
     final ThreadLocal<String> explicitTag = new ThreadLocal<>();
 
-    String getTag() {
+    private String getTag() {
```

Comment:
This was probably package private to remove the creation of synthetic accessor methods.